packages/story-editor/src/components/form/advancedDropDown/container.js

```diff
@@ -40,7 +40,7 @@ const Container = styled.div`
   flex-direction: row;
   flex-wrap: wrap;
   width: 100%;
-  min-width: 160px;
+  min-width: 120px;
```

Comment:
Q: Does this change influence any other use cases of the advanced dropdown?

Reply:
We use it for font picker and authors, both are safe (they are 100% width, 276px most of the time).